docs/tool-configuration/configuration-helm.md

```diff
@@ -104,12 +104,12 @@ will skip helm execution
 sets helm release name
 
 -------------------
-### uninstall-charts
-* **BitOps Property:** `uninstall-charts`
-* **Environment Variable:** `HELM_UNINSTALL_CHARTS`
+### uninstall
+* **BitOps Property:** `uninstall`
+* **Environment Variable:** `HELM_UNINSTALL`
 * **default:** `""`
 
-Comma separated string. If any of the charts to be deployed match one of the chart names listed here, it will be uninstalled with `helm uninstall $HELM_RELEASE_NAME` instead of deployed/upgraded.
+If true, this chart will be uninstalled instead of deployed/upgraded. If the environment variable `HELM_UNINSTALL` is passed in to the container, all BitOps managed charts for a given environment will be uninstalled.
```

Comment:
can we do `HELM_UNINSTALL_CHARTS` as an env variable to allow selectively uninstalling specific charts?

this would be in addition to a `bitops.config.yaml` option per chart of `uninstall: true`

?

Reply:
It still exists https://github.com/bitovi/bitops/pull/36/files#diff-4d10912c465f98526cdbff10a8bda81b398b22e7007779eaad7f36dc8b562b6dR179-R180